libs/akita/src/lib/queryEntity.ts

```diff
@@ -291,22 +291,28 @@ export class QueryEntity<S extends EntityState, EntityType = getEntityType<S>, I
    * Listen for entity actions
    *
    *  @example
-   *
    *  this.query.selectEntityAction(EntityActions.Add);
    *  this.query.selectEntityAction(EntityActions.Update);
    *  this.query.selectEntityAction(EntityActions.Remove);
    *
+   *  this.query.selectEntityAction([EntityActions.Add, EntityActions.Update, EntityActions.Remove])
+   *
    *  this.query.selectEntityAction();
    */
   selectEntityAction(action: EntityActions): Observable<IDType[]>;
+  selectEntityAction(actions: EntityActions[]): Observable<IDType[]>;
```

Comment:
Should be `  selectEntityAction(actions: EntityActions[]): Observable<EntityAction<IDType>>;`